.buildkite/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "kibana-buildkite",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
this and a lot of removals happened from a seemingly benign addition of a package - was this expected? is it a new `npm` version that caused this?